plugins/standard/applets/conference/conference-calls.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+if(isset($_GET['json'])):
+
+	$ci =& get_instance();
+
+	$service = new Services_Twilio($ci->twilio_sid, $ci->twilio_token);
+
+	$conferences = $service->account->conferences->getIterator(0, 50, array("Status" => "in-progress"));
+
+	$res = array();
+	
+	foreach($conferences as $call) {
+		$res[$call->friendly_name] = array(
+			'date_created' => date("F j, Y, g:i a",strtotime($call->date_created)),
+			'friendly_name' => $call->friendly_name,
+			'status' => $call->status,
+			'duration' => $call->duration
+		);		
+	}
+	
+	die(json_encode($res));
+	
+endif;
+
+?>
+
+<script type="text/javascript" src="//ajax.googleapis.com/ajax/libs/jquery/1.2.6/jquery.min.js"></script>
+<script>
+
+	function join_call(conference_name){
+		
+		window.parent.Client.call({ 
+			'conference_name' : conference_name,
+			'muted' : 'false',
+			'beep' : 'true',
+			'Digits' : 1});
+		
+		return false;
+	}
+	
+	function listen_call(conference_name){
+
+		window.parent.Client.call({ 
+			'conference_name' : conference_name,
+			'muted' : 'true',
+			'beep' : 'false',
+			'Digits' : 1});
+		
+		return false;
+	}
+	
+	var $ = jQuery;
+		
+	$(function(){
+				
+		var
+		calls = {},
+		select = $('#calls'),
+		updateCalls = function() {
+			$.getJSON(window.location + '/?json=1', function(data) {
+				$.each(data, function(conference_name, call) {
+					if(!calls[conference_name]) {
+						calls[conference_name] = call;
+						select.append('<tr class="message-row recording-type" id="' + call.friendly_name + '"><td class="recording-date">' + call.date_created + '</td><td class="recording-duration">' + call.friendly_name + '</td><td class="recording-duration">' + call.status + '</td><td class="recording-duration" ><a id="join" onclick="join_call(\'' + call.friendly_name + '\');">Join</a></td><td class="recording-duration" ><a id="listen" onclick="listen_call(\'' + call.friendly_name + '\');">Listen</a></td></tr>');
```

Comment:
This will add multiple items to the DOM with `id="join"` and `id="listen"`. HTML id attributes should be unique. You can convert them to classes and not lose anything here.

Should also break this out to multiple lines for readability.